utils/travis/steps/install.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+date
```

Comment:
What's the point of this? Temporary test output you forgot to remove?

Reply:
I had two builds queued at one point, but for some reason the second xcode job didn't pick up on the cache that should've been present from the first build.  It would have been useful to be able to tell if the xcode job in the second build had started before the xcode job in the first build ended.

Reply:
OK. You can keep this line, then.